app/components/provider_interface/completed_offer_summary_component.rb

```diff
@@ -58,5 +66,9 @@ def change_location_path
     def mode
       :edit
     end
+
+    def qualification_text
+      course.qualifications.reverse.map(&:upcase).join(' with ')
```

Comment:
This is reused across all components, maybe we could move this into a helper method, and test different variations there?

Reply:
Are qualifications always the same, would we ever encounter a different new variation that doesn't match this format and need to be more explicit, also what should we show when this value is `nil`?

Reply:
spoke to adam and we will hide the row if it is a `nil` value